File: DotNet/PopulationFitness/PopulationFitness/Tuning.cs
using PopulationFitness.Models.Genes;
using System;

namespace PopulationFitness
{
    public class Tuning
    {
        public String Id;
        public Function Function = Function.Undefined;
        public double HistoricFit = 1.0;
        public double DiseaseFit = 50;
        public double ModernFit = 0.01;
        public double ModernBreeding = 0.07;
        public int NumberOfGenes = 10;
        public int SizeOfGenes = 4;
        public double MutationsPerGene = 0.2;
        public int SeriesRuns = 1;
        public int ParallelRuns = 1;
    }
}

File: Scala/PopulationFitness/src/main/scala/uk/edu/populationfitness/models/tuning/Tuning.scala
package uk.edu.populationfitness.models.tuning

import uk.edu.populationfitness.models.genes.FitnessFunction
import uk.edu.populationfitness.models.genes.FitnessFunction.FitnessFunction

class Tuning {
  var id: String = null
  var function: FitnessFunction = FitnessFunction.Undefined
  var historic_fit: Double = 1.0
  var disease_fit: Double = 50.0
  var modern_fit: Double = 0.01
  var modern_breeding: Double = 0.07
  var number_of_genes:Int = 10
  var size_of_genes:Int = 4
  var mutations_per_gene: Double = 0.2
  var series_runs:Int = 1
  var parallel_runs:Int = 1
}